### ISSUE TITLE
Can't change to beta channel [Unrecognized flags: disable-dart-dev]

### ISSUE THREAD

**Issue by devibrahimkarahan** [flutter/flutter#60202]
Hi everyone,

I cant change flutter channel to beta from stable. My Flutter version is 1.17.4

Here is `flutter doctor -v` when channel is stable.
![image](https://user-images.githubusercontent.com/7039967/85563419-09600b80-b636-11ea-92ed-026c85a2ca6a.png)

All flutter commands works when channel is stable, here is `flutter channel`
![image](https://user-images.githubusercontent.com/7039967/85563969-88554400-b636-11ea-86ba-d32c61aeaada.png)

When i type `flutter channel beta`, here is the output
![image](https://user-images.githubusercontent.com/7039967/85564553-0b769a00-b637-11ea-845c-3795a63f5a4c.png)

Now all flutter commands give me same error, here is `flutter upgrade`
![image](https://user-images.githubusercontent.com/7039967/85564739-2f39e000-b637-11ea-9875-a67ff0f61f6d.png)

I cant get `flutter doctor -v` log cuz happening same error.

**Comment by iapicca** [comment]
Hi @devibrahimkarahan 
could you please make sure that you meet the [system requirements](https://flutter.dev/docs/get-started/install/windows#system-requirements),
if so
please run in your terminal 
(posting the output as text or .txt file and not pictures)
```console
flutter doctor -v
flutter channel beta
flutter upgrade --force -v
flutter doctor -v
```
Thank you

**Comment by devibrahimkarahan** [comment]
Hi,

Here is first `flutter doctor -v`

```
C:\src\flutter\bin>flutter doctor -v
Checking Dart SDK version...
Flutter requires PowerShell 5.0 or newer.
See https://flutter.dev/docs/get-started/install/windows for more.
Building flutter tool...
Running pub upgrade...
[ÔêÜ] Flutter (Channel stable, v1.17.4, on Microsoft Windows [Version 6.3.9600],

    locale en-US)
    ÔÇó Flutter version 1.17.4 at C:\src\flutter
    ÔÇó Framework revision 1ad9baa8b9 (7 days ago), 2020-06-17 14:41:16 -0700
    ÔÇó Engine revision ee76268252
    ÔÇó Dart version 2.8.2

[ÔêÜ] Android toolchain - develop for Android devices (Android SDK version 29.0.
2)
    ÔÇó Android SDK at C:\Users\IKarahan\AppData\Local\Android\sdk
    ÔÇó Platform android-29, build-tools 29.0.2
    ÔÇó Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    ÔÇó Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b03)
    ÔÇó All Android licenses accepted.

[ÔêÜ] Android Studio (version 3.5)
    ÔÇó Android Studio at C:\Program Files\Android\Android Studio
    ÔÇó Flutter plugin version 43.0.1
    ÔÇó Dart plugin version 191.8593
    ÔÇó Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b03)

[ÔêÜ] IntelliJ IDEA Community Edition (version 2019.2)
    ÔÇó IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA Community Edition
      2019.2
    ÔÇó Flutter plugin version 46.0.1
    ÔÇó Dart plugin version 192.8052

[!] VS Code (version 1.46.1)
    ÔÇó VS Code at C:\Users\IKarahan\AppData\Local\Programs\Microsoft VS Code
    X Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.
```

`flutter channel beta`
```
C:\src\flutter\bin>flutter channel beta
Checking Dart SDK version...
Flutter requires PowerShell 5.0 or newer.
See https://flutter.dev/docs/get-started/install/windows for more.
Building flutter tool...
Running pub upgrade...
Switching to flutter channel 'beta'...
git: Branch 'beta' set up to track remote branch 'beta' from 'origin'.
git: Switched to a new branch 'beta'
Successfully switched to flutter channel 'beta'.
To ensure that you're on the latest build from this channel, run 'flutter
upgrade'
```

`flutter upgrade --force -v`
```
C:\src\flutter\bin>flutter upgrade --force -v
Checking Dart SDK version...
Flutter requires PowerShell 5.0 or newer.
See https://flutter.dev/docs/get-started/install/windows for more.
Building flutter tool...
Running pub upgrade...
Setting VM flags failed: Unrecognized flags: disable-dart-dev
```

`flutter doctor -v`
```
C:\src\flutter\bin>flutter upgrade --force -v
Checking Dart SDK version...
Flutter requires PowerShell 5.0 or newer.
See https://flutter.dev/docs/get-started/install/windows for more.
Building flutter tool...
Running pub upgrade...
Setting VM flags failed: Unrecognized flags: disable-dart-dev
```

**Comment by devibrahimkarahan** [comment]
After many trials, including reinstall flutter, I solved it with reinstall **Windows PowerShell 5.1** last release.
